evaluation/numpy/segmentation_and_tracking_quality.py

```diff
@@ -122,9 +176,12 @@ def update_state(self, y_true: np.ndarray, y_pred: np.ndarray, sequence_id=0):
       sequence_id: The optional ID of the sequence the frames belong to. When no
         sequence is given, all frames are considered to belong to the same
         sequence (default: 0).
+      weights: The weights for each pixel with the same shape of `y_true`.
```

Comment:
Minor: `The optional weights ... Defaults to using the same weight for every pixel.`